_examples/sse/webapp/client.gen.ts

```diff
@@ -1,18 +1,69 @@
 /* eslint-disable */
-// webrpc-sse-chat v1.0.0 e64bf9372e2faa45da9d449e8ea108dc0597a98d
+// webrpc-sse-chat v1.0.0 a799dc63b082644f5d003c8881424546aee23a2c
 // --
-// Code generated by webrpc-gen@v0.19.3 with ../../ generator. DO NOT EDIT.
+// Code generated by webrpc-gen@v0.20.3-2-g18c094a with ../../ generator. DO NOT EDIT.
 //
 // webrpc-gen -schema=service.ridl -target=../../ -client -out=./webapp/client.gen.ts
 
+export const WebrpcHeader = "Webrpc"
+
+export const WebrpcHeaderValue = "webrpc@v0.20.3-2-g18c094a;@unknown;webrpc-sse-chat@v1.0.0"
+
 // WebRPC description and code-gen version
 export const WebRPCVersion = "v1"
 
 // Schema version of your RIDL schema
 export const WebRPCSchemaVersion = "v1.0.0"
 
 // Schema hash generated from your RIDL schema
-export const WebRPCSchemaHash = "e64bf9372e2faa45da9d449e8ea108dc0597a98d"
+export const WebRPCSchemaHash = "a799dc63b082644f5d003c8881424546aee23a2c"
+
+type WebrpcGenVersions = {
+  webrpcGenVersion: string;
+  codeGenName: string;
+  codeGenVersion: string;
+  schemaName: string;
+  schemaVersion: string;
+};
+
+export function VersionFromHeader(headers: Headers): WebrpcGenVersions | null {
+  const headerValue = headers.get(WebrpcHeader);
+  if (!headerValue) {
+    throw new Error("header is empty or missing");
```

Comment:
Do we want to throw here? Isn't an early return and a console.error better suited